packages/vue/src/components/range/RangeSlider.jsx

```diff
@@ -32,6 +32,10 @@ const RangeSlider = {
 			start: VueTypes.integer.def(0),
 			end: VueTypes.integer.def(10),
 		}),
+		rangeLabels: VueTypes.shape({
```

Comment:
Do we not have this prop type in `types` list?

Reply:
@bietkul I guess `rangeLabels`  prop is only for `rangeSliders` we don't have this in `types` list, we have `missingLabel` and `selectAllLabel` but they are used when we select any `filter`.